websites/M/MultiplayerPiano.net/presence.ts

```diff
@@ -0,0 +1,54 @@
+const presence = new Presence({
+	clientId: "1240257888514080830",
+});
+
+const enum Assets { // Other default assets can be found at index.d.ts
+	Logo = "https://cdn.discordapp.com/attachments/587336926533648409/1240591699034767380/mpp2.png?ex=66471e8e&is=6645cd0e&hm=b288e175fe293d96c41c4041773d370a231a52a4cdd818eaa88342731a6762f9&",
```

Comment:
These Discord links are temporary and may not work. The link in the metadata should be fine.
```suggestion
	Logo = "https://www.virtualdrumming.com/icons-piano/icons-512.png",
```